libethereum/BlockChain.cpp

```diff
@@ -1535,7 +1535,7 @@ VerifiedBlockRef BlockChain::verifyBlock(bytesConstRef _block, std::function<voi
 			try
 			{
 				Transaction t(d, (_ir & ImportRequirements::TransactionSignatures) ? CheckTransaction::Everything : CheckTransaction::None);
-				m_sealEngine->verifyTransaction(_ir, t, h);
+				m_sealEngine->verifyTransaction(_ir, t, EnvInfo(h));
```

Comment:
we use m_sealEngine->verifyTransaction in State where there are no blockheader. so not to construct fake blockheader at there Yann replaced the function to accept EnvInfo (which is used at State)